src/main/resources/application.yml

```diff
@@ -0,0 +1,71 @@
+spring:
```

Comment:
Please provide just one file per profile.  I mean, application-dev , application-prod. Leave application.yml just for general common configurations